packages/orchestration/src/cosmos-api.ts

```diff
@@ -317,17 +328,15 @@ export interface IBCMsgTransferOptions {
  *
  * @see {OrchestrationAccountI}
  */
-export type CosmosChainAccountMethods<CCI extends CosmosChainInfo> =
+export type CosmosChainAccountMethods<CCI extends CosmosChainInfo> = Pick<
+  IcaAccount,
+  'deactivate' | 'executeEncodedTx' | 'reactivate'
+> &
   (CCI extends {
-    icaEnabled: true;
```

Comment:
`CosmosChainAccountMethods` now assumes that ICA methods are available for every value in `CosmoChainInfo`, regardless of of whether they have ICA enabled. Why is this legitimate?